package/linux-pam/0005-faillock-remove-user-resolution.patch

```diff
@@ -0,0 +1,26 @@
+--- a/modules/pam_faillock/pam_faillock.c
++++ b/modules/pam_faillock/pam_faillock.c
+@@ -374,6 +374,10 @@
+ 		return PAM_IGNORE;
+ 	}
+ 
++ 	// siklu changes for Jira ticket PORTF-1547:
++	//  - user from TACACS/RADIUS server cannot be resolved (user-name -> uid) here
++	//  - hardcode output values
++#if 0
+ 	if ((pwd=pam_modutil_getpwnam(pamh, user)) == NULL) {
+ 		if (opts->flags & FAILLOCK_FLAG_AUDIT) {
+ 			pam_syslog(pamh, LOG_NOTICE, "User unknown: %s", user);
+@@ -395,7 +399,11 @@
+ 		opts->is_admin = pam_modutil_user_in_group_uid_nam(pamh,
+ 			pwd->pw_uid, opts->admin_group);
+ 	}
+-
++#endif
++	(void)pwd;
++	opts->user = user; 	// will be used as filename for tally file
++	opts->uid = 0;		// will be used as uid of tally file
```

Comment:
this will render user as root 
as i understand ATM it is just placehoslder with no any consequences, but better bind it to some other value. `500` (local admin) at least, or `1000` (first custom user id)

Reply:
fixed